apl-core/src/main/java/com/apollocurrency/aplwallet/apl/core/rest/service/ServerInfoService.java

```diff
@@ -135,7 +135,11 @@ public ApolloX509Info getX509Info(){
         List<Generator> forgers = Generator.getSortedForgers();
         for (Generator g : forgers) {
             GeneratorInfo gi = new GeneratorInfo();
-            gi.setAccount(Convert.defaultRsAccount(g.getAccountId()));
+            gi.setAccount(
+                Convert.parseAccountId(
+                    Convert.defaultRsAccount(g.getAccountId())
+                )
```

Comment:
Double conversion is unnecessary here,
we can just use gi.setAccount(g.getAccountId())

Reply:
done